.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,56 @@
+name: CMock Unit Tests
+on: [push]
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Parent Repo
+      uses: actions/checkout@v2
+      with:
+        ref: master
+        repository: FreeRTOS/FreeRTOS
+        submodules: 'recursive'
+        fetch-depth: 1
+    - name: Clone This Repo
+      uses: actions/checkout@v2
+      with:
+        path: ./FreeRTOS/Source
+
+    - name: Setup Python
+      uses: actions/setup-python@master
+      with:
+        python-version: 3.8
```

Comment:
This is a vulnerable python verison. Please use 3.7.10 or a patched python version

Reply:
"3.8" maps to the latest 3.8.X version, which is 3.8.8 and includes the fix for CVE-2021-3177.
Example run: https://github.com/paulbartell/FreeRTOS-Kernel/runs/2125390265?check_suite_focus=true
Release notes for 3.8.8: https://www.python.org/downloads/release/python-388/